servers/rendering/shader_language.cpp

```diff
@@ -3617,6 +3635,13 @@ bool ShaderLanguage::_parse_function_arguments(BlockNode *p_block, const Functio
 			_set_error(RTR("Expected ',' or ')' after argument."));
 			return false;
 		}
+		// Peek at whether there's a closing parenthesis after the last comma;
+		// if so, it must be a argument-list-trailing comma.
+		if (_peek_token().type == TK_PARENTHESIS_CLOSE) {
+			// Swallow the closing parenthesis and return.
+			ERR_FAIL_COND_V(_get_token().type != TK_PARENTHESIS_CLOSE, false);
+			break;
+		}
```

Comment:
I'm not sure this sequence makes sense, is it actually possible for the peeked token to not end up being the correct token? If `_peek_token` can result in a different result than `_get_token` I'd say something is wrong

Reply:
We can do that. Just feels incorrect to get the token and not do anything about it.